### PR TITLE
i18n and RTL improvements

### PR DESCRIPTION
**Pull request by eranroz**
- Use mw-title and mw-userlink to fix bidi issues
  (MW css classes set it as unicode-bidi: embed)
- Extracting local strings to messages

### CODE REVIEW

**Comment by eranroz** [comment]
I'm not familiar with intuition - so please check it carefully + add the relevant messages

**Comment by Krinkle** [comment]
Some of these issues have been fixed by 91a9d664aa56fa6e8faf6d9fe15929613ba299aa (more localisation messages) and 557c53910a6fac07f677b8c1b472224c087e1379 (mw-userlink).

**Comment by eranroz** [comment]
Dependency: https://github.com/Krinkle/intuition/pull/61

File: src/rtrc.js [hunk 1]
```diff
@@ -219,7 +219,7 @@
 		itemClass = [];
 
 		if (rc.type === 'new') {
-			typeSymbol += '<span class="newpage">N</span>';
+			typeSymbol += '<span class="newpage">' + mw.msg('newpageletter') + '</span>';
```
Comment (Krinkle) [inline src/rtrc.js]: This may need escaping. Use `mw.message().escaped()` for consistency with the rest.

File: src/rtrc.js [hunk 2]
```diff
@@ -1305,8 +1305,8 @@ Example:
 						'<h3>' + title + '</h3>' +
 						'<div class="mw-rtrc-diff-tools">' +
 							'<span class="tab"><a id="diffClose">X</a></span>' +
-							'<span class="tab"><a href="' + href + '" target="_blank" id="diffNewWindow">Open in Wiki</a></span>' +
-							'<span class="tab"><a onclick="$(\'.patrollink a\').click()">[mark]</a></span>' +
+							'<span class="tab"><a href="' + href + '" target="_blank" id="diffNewWindow">' + message('openInWiki').escaped() + '</a></span>' +
```
Comment (Krinkle) [inline src/rtrc.js]: Message keys should all be in lowercase.  The intuition commit refers to it as `open-in-wiki`.

File: src/rtrc.js [hunk 3]
```diff
@@ -1466,6 +1466,7 @@ Example:
 					'markedaspatrollederror',
 					'namespaces',
 					'namespacesall',
+					'newpageletter',
```
Comment (Krinkle) [inline src/rtrc.js]: Nice 👍

**Comment by Krinkle** [comment]
I extracted the mw-title class addition and landed it (f2ff74a8).

**Pull request closed without merging**